types/payments.ts

```diff
@@ -344,33 +344,32 @@ export interface PatchPaymentRequest {
     }
 }
 
-export type CreatePaymentRequest = CreateWirePaymentRequest | CreateBookPaymentRequest | CreateInlinePaymentRequest | CreateLinkedPaymentRequest | CreateVerifiedPaymentRequest
+export type CreatePaymentRequest = CreateWirePaymentRequest | CreateBookPaymentRequest | CreateInlinePaymentRequest | CreateLinkedPaymentRequest | CreateVerifiedPaymentRequest | CreatePushToCardPaymentRequest
 
 export interface PushToCardPayment {
     /**
-     * The amount (cents) of the payment.
-     */
-    amount: number
-  
-    /**
-     * Payment description (maximum of 50 characters), this will show up on statement of the counterparty.
-     */
-    description: string
-  
-    /**
-     * Provider configuration, see [Configuration](https://docs.unit.co/types#push-to-card-configuration)
-     */
-    configuration: PushToCardConfiguration
-  
-    /**
-     * See Idempotency(https://docs.unit.co/#intro-idempotency)
+     * Identifier of the received payment resource.
      */
-    idempotencyKey?: string
-  
+    id: string
+
     /**
-     * See [Tags](https://developers.unit.co/#tags)
+     * Type of the transaction resource. The value is always pushToCardPayment.
      */
-    tags?: Tags
+    type: "pushToCardPayment"
+
+    attributes: BasePaymentAttributes & {
```

Comment:
```suggestion
    attributes: Omit<BasePaymentAttributes, 'direction'> & {
```

https://docs.unit.co/resources#push-to-card-payment - the push to card payment has no direction

Reply:
I see, I was looking at the JSON and I saw that it was just defaulted to Credit, so I kept it